core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -52,14 +52,16 @@ oppia.directive('tutorCard', [
         '$scope', '$timeout', 'oppiaPlayerService', 'HintManagerService',
         'playerPositionService', 'playerTranscriptService',
         'ExplorationPlayerStateService', 'windowDimensionsService',
-        'urlService', 'TWO_CARD_THRESHOLD_PX', 'CONTENT_FOCUS_LABEL_PREFIX',
+        'urlService', 'AudioPlayerService', 'AudioTranslationManagerService',
```

Comment:
removed, except for AudioPlayerService which I use now for stopping audio on changing cards